CONTRIBUTING.md

```diff
@@ -25,3 +25,14 @@ Open PRs without a reviewer can be treated as a draft. When a reviewer is added
 
 
 [tox]: https://tox.readthedocs.io/en/latest/
+
+## Contributing documentation
+To test contributions to the documentation, this can be built locally following:
+
+    python -m pip install ".[docs]"
+    sphinx-build -W -b html docs docs/_build/html
+
+Python's built in http server can be then used to serve the page:
+
+    cd docs/_build/html
+    python -m http.server
```

Comment:
You can also just `open docs/_build/html/index.html` on Mac.

Reply:
Ah, nice! I'll add that in. Maybe the above is more universal?